WordPress/Classes/ViewRelated/Reader/Controllers/ReaderSearchViewController.swift

```diff
@@ -62,8 +63,11 @@ final class ReaderSearchViewController: UIViewController {
     override func viewDidAppear(_ animated: Bool) {
         super.viewDidAppear(animated)
 
-        DispatchQueue.main.async {
-            self.searchController.searchBar.becomeFirstResponder()
+        if isFirstAppearance {
```

Comment:
Is `isMovingToParent` applicable here?

Reply:
I don't know. I trust `viewDidAppear(_ animated: Bool)` and `isFirstAppearance` more.